src/main/scala/com/databricks/spark/redshift/Utils.scala

```diff
@@ -93,7 +93,9 @@ private[redshift] object Utils {
       val key = Option(s3URI.getKey).getOrElse("")
       Option(s3Client.getBucketLifecycleConfiguration(bucket)) match {
         case None =>
-          log.error(s"The S3 bucket $bucket does not exist")
+          log.warn(s"No configuration were found for S3 bucket $bucket. " +
+             "Please make sure that the bucket exists and it has a Life Cycle " +
```

Comment:
Can you please try to match the writing style of the existing messages? "Life Cycle" should be "lifecycle" and this sentence should end with a period.